app/dashboard/page.tsx

```diff
@@ -0,0 +1,13 @@
+import Link from 'next/link'
+
+export default function Home() {
```

Comment:
Eventhough it is a default export the name of the component should reflect the page intent

Reply:
Commets added apply for all files

Reply:
Done